data-structures/linkedList/linked-list.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+class Node {
+  constructor(val) {
+    this.val = val;
+    this.next = null;
+    this.prev = null;
+  }
+}
+
+class LinkedList {
+  constructor() {
+    this.head = null;
+  }
+
+  insert(val) {
+    try {
+      let newNode = new Node(val);
+  
+      newNode.next = this.head;
+      this.head = newNode;
+
+    } catch 
+  }
+
+  // includes(currVal) {
+  //     let currNode = this.head;
+
+  //     while(currNode != null) 
+
```

Comment:
while it transverse through the linkedlist your going to check the _value_ of each node to the _currVal_ and return true or keep going till it reaches null then return false.